Demo/src/main/java/com/paypal/android/uishared/components/StepHeader.kt

```diff
@@ -0,0 +1,77 @@
+package com.paypal.android.uishared.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.PlatformTextStyle
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.paypal.android.utils.UIConstants
+
+@Composable
+fun StepHeader(
+    stepNumber: Int,
+    title: String,
+    modifier: Modifier = Modifier
```

Comment:
Do we need this modifier param? I don't see it used in this PR anywhere

Reply:
Yeah technically we aren't using it, but if we ever wanted to apply styles like padding, etc. from from the parent element we could do so. I saw it in a lot of the sample projects Google has, but you're right it doesn't have much use in our project at the moment. I'm down to take or leave.

Reply:
Went ahead and took it out 👍